test/aqua/operators/test_op_construction.py

```diff
@@ -447,6 +447,19 @@ def test_op_indent(self, op):
         ).split("\n{}".format(op.INDENTATION))
         self.assertListEqual(indented_str_content, initial_str.split("\n"))
 
+class TestOpMethods(QiskitAquaTestCase):
+    """Basic method tests."""
+
+    def test_listop_num_qubits(self):
+        op = ListOp([X ^ Y, Y ^ Z])
+        self.assertEqual(op.num_qubits, 2)
+        op = ListOp([X ^ Y, Y])
+
+        with self.assertRaises(ValueError):
+            op.num_qubits
+
+        with self.assertRaises(ValueError):
+            X @ op
```

Comment:
Could you wrap these in sub-tests, one for the ListOp with equal number of qubits and one for the other one?

Reply:
(somehow missed this comment earlier!) Yes.
It might be a good idea to test that `op @ X` (as opposed to `X @ op`) does *not* raise an error.